nautobot_golden_config/nornir_plays/config_backup.py

```diff
@@ -46,7 +46,11 @@ def run_backup(  # pylint: disable=too-many-arguments
         result (Result): Result from Nornir task
     """
     obj = task.host.data["obj"]
-    settings = device_to_settings_map[obj.id]
+    try:
+        settings = device_to_settings_map[obj.id]
+    except KeyError:
+        logger.log_failure(obj, f"{obj.name} is not in scope so will be skipped.")
```

Comment:
How about something like: 

logger.log_failure(obj, f"There is critical issue with your data, somehow the device `{obj.name}` is being attempted to run, despite it not being found in any of the scopes. Please contact your admin to remedy this. Until the data has been cleaned up, you will receive this message.")

From my understanding of the issue, this better reflects what is going on.

Reply:
Works for me. I was just trying to be concise and not spam too much at the user.

Reply:
This has been updated.